flink-runtime/src/test/java/org/apache/flink/runtime/heartbeat/TestingHeartbeatServices.java

```diff
@@ -18,43 +18,18 @@
 
 package org.apache.flink.runtime.heartbeat;
 
-import org.apache.flink.runtime.clusterframework.types.ResourceID;
 import org.apache.flink.runtime.concurrent.ScheduledExecutor;
-import org.apache.flink.runtime.testingUtils.TestingUtils;
-import org.apache.flink.util.Preconditions;
-
-import org.slf4j.Logger;
 
 /**
  * A {@link HeartbeatServices} that allows the injection of a {@link ScheduledExecutor}.
  */
 public class TestingHeartbeatServices extends HeartbeatServices {
 
-	private final ScheduledExecutor scheduledExecutorToUse;
-
-	public TestingHeartbeatServices(long heartbeatInterval, long heartbeatTimeout, ScheduledExecutor scheduledExecutorToUse) {
+	public TestingHeartbeatServices(long heartbeatInterval, long heartbeatTimeout) {
 		super(heartbeatInterval, heartbeatTimeout);
-
-		this.scheduledExecutorToUse = Preconditions.checkNotNull(scheduledExecutorToUse);
 	}
 
 	public TestingHeartbeatServices() {
-		this(1000L, 10000L, TestingUtils.defaultScheduledExecutor());
-	}
-
-	@Override
-	public <I, O> HeartbeatManager<I, O> createHeartbeatManagerSender(
-		ResourceID resourceId,
-		HeartbeatListener<I, O> heartbeatListener,
-		ScheduledExecutor mainThreadExecutor,
-		Logger log) {
-
-		return new HeartbeatManagerSenderImpl<>(
-			heartbeatInterval,
-			heartbeatTimeout,
-			resourceId,
-			heartbeatListener,
-			scheduledExecutorToUse,
-			log);
+		this(1000L, 10000L);
```

Comment:
seems this class is not needed any more or at least the comment is outdated

Reply:
I'll double check.